apps/insights/package.json

```diff
@@ -33,16 +35,21 @@
     "bs58": "catalog:",
     "change-case": "catalog:",
     "clsx": "catalog:",
+    "color": "^5.0.3",
```

Comment:
We should stick to catalog dependencies unless you actually have a good reason to lock to a lower dep version for some package, makes updating things easier